2019-12-27 18:10:40 INFO  ConnectionUtil:21 - Connected to database.
2019-12-27 18:45:17 INFO  ConnectionUtil:40 - Connected to database.
2019-12-28 17:14:51 INFO  ConnectionUtil:37 - Connected to database.
2019-12-28 17:16:53 INFO  ConnectionUtil:37 - Connected to database.
2019-12-28 17:17:49 INFO  ConnectionUtil:37 - Connected to database.
2019-12-28 17:21:39 INFO  ConnectionUtil:37 - Connected to database.
2019-12-28 17:23:51 INFO  ConnectionUtil:37 - Connected to database.
2019-12-28 17:25:47 INFO  ConnectionUtil:37 - Connected to database.
2019-12-28 17:26:39 INFO  ConnectionUtil:37 - Connected to database.
2019-12-28 17:29:08 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 18:46:17 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 18:46:17 WARN  UserDAOImpl:81 - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
2020-01-03 18:57:49 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 18:57:49 WARN  UserDAOImpl:82 - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
2020-01-03 18:59:15 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 18:59:15 WARN  UserDAOImpl:82 - org.postgresql.util.PSQLException: Operation requires a scrollable ResultSet, but this ResultSet is FORWARD_ONLY.
2020-01-03 19:00:24 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 19:00:24 WARN  UserDAOImpl:82 - org.postgresql.util.PSQLException: ResultSet not positioned properly, perhaps you need to call next.
2020-01-03 19:02:41 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 19:02:41 WARN  UserDAOImpl:82 - org.postgresql.util.PSQLException: The column name users_id was not found in this ResultSet.
2020-01-03 19:03:29 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 19:03:29 WARN  UserDAOImpl:82 - org.postgresql.util.PSQLException: ERROR: relation "p1.employee" does not exist
  Position: 15
2020-01-03 19:05:57 INFO  ConnectionUtil:37 - Connected to database.
2020-01-03 19:05:57 WARN  UserDAOImpl:82 - org.postgresql.util.PSQLException: ERROR: relation "p1.employee" does not exist
  Position: 15
2020-01-03 19:09:31 INFO  ConnectionUtil:37 - Connected to database.
